README.md

```diff
@@ -1,56 +1,101 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Budget Tracker
 
-> Description the project.
+- This is a mobile web application where you can manage your budget, you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+### login
+
+![image](https://user-images.githubusercontent.com/98453979/188453030-26b5fe51-8d89-4d11-9fcc-36fe69c754ba.png)
+
+### homepage
+
+![image](https://user-images.githubusercontent.com/98453979/188453598-17482b66-12e1-418e-873e-d67bf1017179.png)
+
+### transactions
+
+![image](https://user-images.githubusercontent.com/98453979/188453788-001a5192-720f-4d32-9f3f-9f3fa6e7ad32.png)
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML & CSS
+- Ruby on Rails
+- Rubocop
+- Rspec
+- OOP
 
-## Live Demo (if available)
+## ERD
 
-[Live Demo Link](https://livedemo.com)
+![image](https://user-images.githubusercontent.com/98453979/188452222-ec0c2c47-24e1-4f1a-a85b-730e7c58b55c.png)
 
 
-## Getting Started
+## Live Demo 
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Budget tracker]( https://powerful-bastion-96834.herokuapp.com)
```

Comment:
- [ ] There seems to be something wrong with your live demo. Whether logging in or signing up, after typing in my details and clicking the submit button, it presents this error page. Kindly fix that.

  ![sth-wrong](https://user-images.githubusercontent.com/36542986/188511750-e93c2193-fa0a-46e9-9843-54f13ace7356.jpg)

Reply:
@TedLivist. Thanks for the review. I deployed the project on Heroku and it worked well but after some time, I got an email from Heroku about maintenance required on my database.  I think that is the cause.

![image](https://user-images.githubusercontent.com/98453979/188512579-628aff00-6eec-473f-9fd0-0cb0d3d5bfb0.png)

Reply:
@frank1738 kindly request another review.

Reply:
Well resolved! ✔️